ui-admin/src/forms/designer/PageDesigner.tsx

```diff
@@ -14,16 +30,59 @@ export type PageDesignerProps = {
 export const PageDesigner = (props: PageDesignerProps) => {
   const { readOnly, value, onChange } = props
 
+  const [showCreatePanelModal, setShowCreatePanelModal] = useState(false)
+
   return (
     <div>
       <h2>Page</h2>
+
+      <div className="mb-3">
+        <Button
+          disabled={readOnly || value.elements.filter(canBeIncludedInPanel).length === 0}
```

Comment:
I would pull `value.elements.filter(canBeIncludedInPanel)` into a variable computed up above.  Also, if it's not too hard, I would include items that are already in another panel in the available questions, so it's easier to rearrange items between panels, rather than having to delete/recreate panels.

Related, this might speak to the need for a QuestionSelector component that takes a current page/panel  and shows the items on that page/panel to choose from, but also allows picking questions from elsewhere in the survey, to allow more complex rearrangement.  (definitely outside the scope of this PR)

Reply:
👍 I'll leave this is as for now because I think this will soon change when exploring your QuestionSelector idea.